docs/framework/migration-guide/net-framework-4-migration-issues.md

```diff
@@ -27,21 +27,21 @@ Assembly: System.Web (in System.Web.dll)
 | **Configuration files** | The root configuration files (the machine.config file and the root Web.config file) for .NET Framework and ASP.NET 4 have been updated to include most of the boilerplate configuration information that was found in the application Web.config files in ASP.NET 3.5. Because of the complexity of the managed IIS 7 and IIS 7.5 configuration systems, running ASP.NET 3.5 applications under ASP.NET 4 and under IIS 7 and IIS 7.5 can result in either ASP.NET errors or IIS errors. | Upgrade ASP.NET 3.5 applications to ASP.NET 4 by using the project upgrade tools in Visual Studio. Visual Studio 2010 automatically modifies the ASP.NET 3.5 application's Web.config file to contain the appropriate settings for ASP.NET 4.<br><br>However, you can run ASP.NET 3.5 applications using .NET Framework 4 without recompilation. In that case, you might have to manually modify the application's Web.config file before you run the application under .NET Framework 4 and under IIS 7 or IIS 7.5. The specific change you must make depends on the combination of software you are working with, including Service Pack (SP) releases. For information about the possible software combinations that are affected by this change and how to resolve problems with specific combinations, see the section "Configuration Errors Related to New ASP.NET 4 Root Configuration" in the document [ASP.NET 4 Breaking Changes](/aspnet/whitepapers/aspnet4/breaking-changes) on the ASP.NET Web site. |
-| **Control rendering** | In previous versions of ASP.NET, some controls emitted markup that you could not disable. By default, this type of markup is no longer generated in ASP.NET 4. The rendering changes affect the following controls:<br><br>* The `Image` and `ImageButton` controls no longer render a `border="0"` attribute.<br>* The `BaseValidator` class and validation controls that derive from it no longer render red text by default.<br>* The `HtmlForm` control does not render a `name` attribute.<br>* The `Table` control no longer renders a `border="0"` attribute.<br><br>Controls that are not designed for user input (for example, the `Label` control) no longer render the `disabled="disabled"` attribute if their `Enabled` property is set to `false` (or if they inherit this setting from a container control). | If you use Visual Studio to upgrade your application from ASP.NET 2.0 or ASP.NET 3.5, the tool automatically adds a setting to the Web.config file that preserves legacy rendering. However, if you upgrade an application by changing the application pool in IIS to target .NET Framework 4, ASP.NET uses the new rendering mode by default. To disable the new rendering mode, add the following setting to the Web.config file:<br><br>`<pages controlRenderingCompatibilityVersion="3.5" />` |
```

Comment:
This file has a lot of "things that look like lists in tables" (but aren't really treated as such by the Markdown parser). Bulk replace of `<br>*` to `<br>\*`.

Reply:
@DavidAnson I agree. I think the original intent was to have lists inside of the tables. What would be a better fix to achieve that?

Reply:
I'd use an actual `<ol>` and `<li>` structure, since lists inside markdown tables aren't automagic

Reply:
Agreed. Once you’re willing to use HTML, building a list like that seems perfectly reasonable.